tests/components/plugwise/test_config_flow.py

```diff
@@ -128,6 +129,26 @@ async def test_form(
     assert len(mock_setup_entry.mock_calls) == 1
     assert len(mock_smile_config_flow.connect.mock_calls) == 1
 
+    entries = hass.config_entries.async_entries(config_flow.DOMAIN)
+    assert entries[0].unique_id == "smile12345"
```

Comment:
```suggestion
    assert result2["result"].unique_id == "smile12345"
```

Reply:
But we don't ask `async_entries`, the created config entry is returned in the config_entries.async_configure.

Reply:
Sorry, misread that, adjusted accordingly